changelogs/fragments/169_add_lineseparator_to_command.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - at - append line-separator to the end of the ``command`` to address the issue #169 (https://github.com/ansible-collections/ansible.posix/issues/169)
```

Comment:
```suggestion
  - at - append line-separator to the end of the ``command``(https://github.com/ansible-collections/ansible.posix/issues/169).
```

Reply:
Thank you for the review. I have fixed it :)